client/db/src/sql/mod.rs

```diff
@@ -1067,11 +1067,11 @@ mod test {
 					.to_str()
 					.unwrap(),
 				create_if_missing: true,
-				cache_size: 204800,
+				cache_size: 20480,
 				thread_count: 4,
 			}),
-			100,
-			0,
+			1,
+			10_000,
```

Comment:
`0` is no timeout

Reply:
I see, then the error we observed is related to some preallocated memory resources in sqlx (potentially the pool size). Tests run in parallel so probably the CI machine is dropping stuff before the test gets to finish. Please whenever you have time revert the timeout change only and see if still green.